tests/cassandra/nodetool.go

```diff
@@ -69,6 +160,9 @@ func (c *Executor) Run(arguments ...string) (string, string, error) {
 		}
 	}
 
+	fmt.Printf("Waiting 10 secs for pod %s to initialize", pod.Name)
+	time.Sleep(10 * time.Second)
```

Comment:
I would prefer some polling here instead of waiting 10 seconds each time, not sure if that's really possible though.

Reply:
Yeah, it can be done by polling the container logs, but I added sleep just to keep it simple as it's only a small script that's being executed. It's given that the container needs time to  run the script while we are immediately running the command after starting the container. WDYT? @ANeumann82 
 fyi this race condition didn't happen in my own konvoy clusters. Tried it multiple times. Only found it in the CI.